### PR TITLE
[DOC] update npu doc

### PR DESCRIPTION
**Pull request by wangjiangben-hw**
## Motivation

Added several model running results on the NPU.

## Modification

1. docs/en/device/npu.md Added several model running results on the NPU.
2. docs/zh_cn/device/npu.md sync npu.md.

### CODE REVIEW

File: docs/en/device/npu.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 ## Usage
 
-Please refer to [link](https://github.com/open-mmlab/mmcv/blob/master/docs/zh_cn/get_started/build.md) installing mmcv on NPU Devices.
+Please refer to [link](https://github.com/open-mmlab/mmcv/blob/master/docs/zh_cn/get_started/build.md) installing mmcv
```
Comment (ZwwWayne) [inline docs/en/device/npu.md:5]: Please refer to the [building documentation of MMCV]() to install MMCV on NPU devices

File: docs/en/device/npu.md [hunk 2]
```diff
@@ -16,20 +17,29 @@ Also, you can use only one NPU to train the model with the following command:
...
+| [retinanet-r18](<>)  |  31.8  |   ---   | [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/retinanet/retinanet_r18_fpn_1x8_1x_coco.py)            | [log](https://download.openmmlab.com/mmdetection/v2.0/npu/retinanet_r18_fpn_1x8_1x_coco.log.json)            |
 | [retinanet-r50](<>)  |  36.6  |   ---   | [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/retinanet/retinanet_r50_fpn_fp16_1x_coco.py)           | [log](https://download.openmmlab.com/mmdetection/v2.0/npu/retinanet_r50_fpn_1x_coco.log.json)                |
+|   [yolov3-608](<>)   |  34.7  |   ---   | [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/yolo/yolov3_d53_fp16_mstrain-608_273e_coco.py)         | [log](https://download.openmmlab.com/mmdetection/v2.0/npu/yolov3_d53_fp16_mstrain-608_273e_coco.log.json)    |
+|  [\*\*yolox-s](<>)   |  39.9  |   ---   | [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/yolox/yolox_s_8x8_300e_coco.py)                        | [log](https://download.openmmlab.com/mmdetection/v2.0/npu/yolox_s_8x8_300e_coco.log.json)                    |
```
Comment (ZwwWayne) [inline docs/en/device/npu.md:30]: We usually put the star behind the item. like yolox-s\*\*

File: docs/en/device/npu.md [hunk 3]
```diff
@@ -16,20 +17,29 @@ Also, you can use only one NPU to train the model with the following command:
...
+| [centernet-r18](<>)  |  26.1  |   ---   | [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/centernet/centernet_resnet18_140e_cocoo.py)            | [log](https://download.openmmlab.com/mmdetection/v2.0/npu/centernet_resnet18_140e_coco.log.jsonn)            |
 |   [\*fcos-r50](<>)   |  36.1  |   ---   | [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/fcos/fcos_r50_caffe_fpn_gn-head_fp16_1x_bs8x8_coco.py) | [log](https://download.openmmlab.com/mmdetection/v2.0/npu/fcos_r50_caffe_fpn_gn-head_1x_coco_bs8x8.log.json) |
 |   [solov2-r50](<>)   |  ---   |  34.7   | [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/solov2/solov2_r50_fpn_1x_coco.py)                      | [log](https://download.openmmlab.com/mmdetection/v2.0/npu/solov2_r50_fpn_1x_coco.log.json)                   |
 
 **Notes:**
 
 - If not specially marked, the results are same between results on the NPU and results on the GPU with FP32.
```
Comment (ZwwWayne) [inline docs/en/device/npu.md:37]:
```suggestion
- If not specially marked, the results on NPU are the same as those on the GPU with FP32.
```

File: docs/en/device/npu.md [hunk 4]
```diff
@@ -16,20 +17,29 @@ Also, you can use only one NPU to train the model with the following command:
...
-- (\*) The results on the NPU of these models are aligned with the results of the mixed-precision training on the GPU, but are lower than the results of the FP32. This situation is mainly related to the phase of the model itself in mixed-precision training, users please adjust the hyperparameters to achieve the best result by self.
+- (\*) The results on the NPU of these models are aligned with the results of the mixed-precision training on the GPU,
+  but are lower than the results of the FP32. This situation is mainly related to the phase of the model itself in
+  mixed-precision training, users please adjust the hyperparameters to achieve the best result by self.
```
Comment (ZwwWayne) [inline docs/en/device/npu.md:40]:
```suggestion
  mixed-precision training, users may need to adjust the hyperparameters to achieve better results.
```

File: docs/en/device/npu.md [hunk 5]
```diff
@@ -16,20 +17,29 @@ Also, you can use only one NPU to train the model with the following command:
...
+- (\*\*) The accuracy of yolox-s on the GPU in mixed precision is 40.1; yolox-s is persister_woker enabled by default,
```
Comment (ZwwWayne) [inline docs/en/device/npu.md:41]:
```suggestion
- (\*\*) The accuracy of yolox-s on the GPU in mixed precision is 40.1, with `persister_woker=True` in the data loader config by default.
```

File: docs/en/device/npu.md [hunk 6]
```diff
@@ -16,20 +17,29 @@ Also, you can use only one NPU to train the model with the following command:
...
+  and there are currently some bugs on NPUs that prevent the last few epochs from running, but the accuracy is less
```
Comment (ZwwWayne) [inline docs/en/device/npu.md:42]:
```suggestion
  There are currently some bugs on NPUs that prevent the last few epochs from running, but the accuracy is less affected and the difference can be ignored.
```

File: docs/en/device/npu.md (hunk 5 above)
Comment (ZwwWayne) [inline docs/en/device/npu.md:41]:
```suggestion
- (\*\*) The accuracy of yolox-s on the GPU in mixed precision is 40.1, with `persister_woker=True` in the data loader config by default.
```

Comment (ZwwWayne) [inline docs/en/device/npu.md:41]:
```suggestion
- (\*\*) The accuracy of yolox-s on the GPU in mixed precision is 40.1, with `persister_woker=True` in the data loader config by default.
```

Comment (ZwwWayne) [inline docs/en/device/npu.md:41]:
```suggestion
- (\*\*) The accuracy of yolox-s on the GPU in mixed precision is 40.1, with `persister_woker=True` in the data loader config by default.
```

File: docs/zh_cn/device/npu.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+| [centernet-r18](<>)  |  26.1  |   ---   | [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/centernet/centernet_resnet18_140e_cocoo.py)            | [log](https://download.openmmlab.com/mmdetection/v2.0/npu/centernet_resnet18_140e_coco.log.jsonn)            |
+|   [\*fcos-r50](<>)   |  36.1  |   ---   | [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/fcos/fcos_r50_caffe_fpn_gn-head_fp16_1x_bs8x8_coco.py) | [log](https://download.openmmlab.com/mmdetection/v2.0/npu/fcos_r50_caffe_fpn_gn-head_1x_coco_bs8x8.log.json) |
+|   [solov2-r50](<>)   |  ---   |  34.7   | [config](https://github.com/open-mmlab/mmdetection/blob/master/configs/solov2/solov2_r50_fpn_1x_coco.py)                      | [log](https://download.openmmlab.com/mmdetection/v2.0/npu/solov2_r50_fpn_1x_coco.log.json)                   |
+
+**Notes:**
+
+- 如果没有特别标记，NPU 上的结果与使用 FP32 的 GPU 上的结果结果相同。
+- (\*) 这些模型在 NPU 上的结果与 GPU 上的混合精度训练结果一致，但低于 FP32 的结果。这种情况主要与模型本身在混合精度训练中的相性有关，
```
Comment (ZwwWayne) [inline docs/zh_cn/device/npu.md:37]:
```suggestion
- (\*) 这些模型在 NPU 上的结果与 GPU 上的混合精度训练结果一致，但低于 FP32 的结果。这种情况主要与模型本身在混合精度训练中的特点有关，
```

File: docs/zh_cn/device/npu.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+  用户请自行调整超参数以达到最佳效果。
```
Comment (ZwwWayne) [inline docs/zh_cn/device/npu.md:38]: 用户可以自行调整超参数来获得更高精度

File: docs/en/device/npu.md [hunk 9]
```diff
@@ -16,20 +17,29 @@ Also, you can use only one NPU to train the model with the following command:
...
-- (\*) The results on the NPU of these models are aligned with the results of the mixed-precision training on the GPU, but are lower than the results of the FP32. This situation is mainly related to the phase of the model itself in mixed-precision training, users please adjust the hyperparameters to achieve the best result by self.
+- If not specially marked, the results on NPU are the same as those on the GPU with FP32.
+- (\*) The results on the NPU of these models are aligned with the results of the mixed-precision training on the GPU,
+  but are lower than the results of the FP32. This situation is mainly related to the phase of the model itself in
+  mixed-precision training, users may need to adjust the hyperparameters to achieve better results.
+- (\*\*) The accuracy of yolox-s on the GPU in mixed precision is 40.1, with `persister_woker=True` in the data loader config by default.
+  There are currently some bugs on NPUs that prevent the last few epochs from running, but the accuracy is less affected and the difference can be ignored.
+  affected and can be ignored.
```
Comment (ZwwWayne) [inline docs/en/device/npu.md:43]: delete this like

**Review by ZwwWayne — APPROVED** [review]

File: docs/en/device/npu.md [hunk 10]
```diff
@@ -2,7 +2,7 @@
 
 ## Usage
 
-Please refer to [link](https://github.com/open-mmlab/mmcv/blob/master/docs/zh_cn/get_started/build.md) installing mmcv on NPU Devices.
+Please refer to the [building documentation of MMCV](https://github.com/open-mmlab/mmdetection/pull/9535) to install MMCV on NPU devices
```
Comment (hhaAndroid) [inline docs/en/device/npu.md:5]: https://github.com/open-mmlab/mmdetection/pull/9535?  The address you can assume has been merged.

**Pull request merged**